laterpay/asset_sources/js/laterpay-backend-pricing.js

```diff
@@ -714,6 +754,32 @@
                         $o.globalDefaultPriceRevenueModelDisplay
                             .text(r.revenue_model_label)
                             .data('revenue', r.revenue_model);
+
+                        if ( 2 === r.post_price_behaviour ) {
+                          $o.lp_current_post_price_val.val('2');
+                          $o.lp_set_inidvidual_price.attr( 'checked', 'checked' );
+                          $o.lp_disable_individual_purchase.removeProp( 'checked' );
+                          $o.lp_make_post_free.removeProp( 'checked' );
+                          $( '#lp_js_globalPriceSection' ).show();
+                          $( '#lp_js_globalRevenueSection' ).show();
+                          $( '#lp_js_formButtons' ).css( 'float', 'none' );
+                        } else if ( 1 === r.post_price_behaviour ) {
+                          $o.lp_current_post_price_val.val('1');
+                          $o.lp_disable_individual_purchase.attr( 'checked', 'checked' );
+                          $o.lp_set_inidvidual_price.removeProp( 'checked' );
+                          $o.lp_make_post_free.removeProp( 'checked' );
+                          $( '#lp_js_globalPriceSection' ).hide();
+                          $( '#lp_js_globalRevenueSection' ).hide();
+                          $( '#lp_js_formButtons' ).css( 'float', 'right' );
+                        } else if ( 0 === r.post_price_behaviour ) {
+                          $o.lp_current_post_price_val.val('0');
+                          $o.lp_make_post_free.attr( 'checked', 'checked' );
+                          $o.lp_set_inidvidual_price.removeProp( 'checked' );
+                          $o.lp_disable_individual_purchase.removeProp( 'checked' );
+                          $( '#lp_js_globalPriceSection' ).hide();
+                          $( '#lp_js_globalRevenueSection' ).hide();
+                          $( '#lp_js_formButtons' ).css( 'float', 'right' );
```

Comment:
Should assign `$( '#lp_js_globalPriceSection' )` to variable and reuse in each if for readablitiy. Same goes for other `$('')` usage.